packages/@aws-cdk/aws-cloudfront/README.md

```diff
@@ -266,9 +270,24 @@ new cloudfront.Distribution(this, 'myDist', {
 });
 ```
 
+> **Note:** The above requires that the stack is in the `us-east-1` region.
+> Lambda@Edge functions must be created in the `us-east-1` region, regardless of the region of the CloudFront distribution and stack.
+> To make it easier to request functions for Lambda@Edge, the `EdgeFunctionExperimental` construct can be used.
+> The `EdgeFunctionExperimental` construct will automatically request a function in `us-east-1`, regardless of the region of the current stack.
+> `EdgeFunctionExperimental` has the same interface as `Function` and can be created and used interchangably.
+
+```ts
+// As an *experimental* replacement, `cloudfront.EdgeFunctionExperimental` can be used in place of `lambda.Function`
+// It will automatically create the `Function` in `us-east-1`, regardless of the current stack's region.
+const myFunc = new cloudfront.EdgeFunctionExperimental(this, 'MyFunction', {
```

Comment:
This can be made to work no problems (and it does not require making a new private/bundled package). You export submodules using the following kind of syntax (typically in `index.ts`):

```ts
export * as experimental from './experimental';
```

Reply:
🪄 magic!